test/fitnesse/responders/run/SuiteResponderTest.java

```diff
@@ -564,7 +570,29 @@ public void loadsCustomFormatters() throws Exception {
 
     assertTrue(FooFormatter.initialized);
   }
+  
```

Comment:
It is a good idea to add the replacement of `.` by `-` in that page names, but I would like to see that in separate test cases where the `fullPathName` actually contains a period and then also just do an `assertEquals` where the expect value is a literal with the periods replaced.

In the current setup the `replace` in the assertion doesn't do anything and the test doesn't make explicit what it is expecting